lang/cs/Org.Apache.REEF.Evaluator.Tests/TaskRuntimeTests.cs

```diff
@@ -63,7 +63,7 @@ public void TestTaskRuntimeInitialization()
         {
             var injector = GetInjector();
             var taskRuntime = injector.GetInstance<TaskRuntime>();
-            Assert.Equal(taskRuntime.GetTaskState(), TaskState.Init);
+            Assert.Equal(taskRuntime.GetTaskState(), EvaluatorTaskState.Init);
```

Comment:
TaskState is defined in both IMRU and Org.Apache.REEF.Common.Runtime.Evaluator.Task. To avoid ambiguous, I defined an alias for Org.Apache.REEF.Common.Runtime.Evaluator.Task.TaskState in this class.